test/integration/logger_test.lua

```diff
@@ -35,8 +35,8 @@ function g.test_cfg()
 
     print()
     g.server:start()
-    g.server:call('package.loaded.log.info', {'Info message'})
-    g.server:call('package.loaded.log.error', {'Error message'})
+    g.server:call('require('log').info', {'Info message'})
```

Comment:
This won't work. 
1) You can't call a function inside net.box:call 
2) This is a syntax error

I suggest leaving those calls as is because there is no hack with changing package functions.

Reply:
OK, reverted changes in `logger_test.lua`.